.github/workflows/builds.yml

```diff
@@ -27,4 +27,4 @@ jobs:
 
       - run: |
           go version
-          go vet ./...
+          go build ./coap-gateway/cmd/service
```

Comment:
Won't this check just the compilation of coap-gw? `go vet ./...` checks if all packages are compilable

Reply:
Only coap-gw is delivered also with golang 1.18 for Kistler.

Reply:
Yes, but I was wondering whether it won't be better to check every binary, but those are checked by other workflows.

Reply:
So we dont'need to check it with go1.18. They are not pushed to ghcr.io